docs/UserGuide.md

```diff
@@ -108,7 +132,7 @@ that you might not remember and organise your tasks neatly.
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 </box>
 
-### Adding a contact : `add_contact`
+## Adding a contact : `add_contact`
 
 Adds a contact to JobFestGo.
 
```

Comment:
I don't think we have to include the `A contact must have mandatory fields name, phone number, email and address`, since in the Features section, we mentioned that the fields without `[]` are mandatory (in line 117)

Reply:
removed